docs/reference-architectures/SharePoint/index.md

```diff
@@ -0,0 +1,553 @@
+---
```

Comment:
path should be all lowercase

docs/reference-architectures/sharepoint/index.md